rancher-vcluster/rancher-vcluster.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rancher-vcluster
```

Comment:
Wondering how will we package this on the harvester side?
or should be allow addon to create the namespace if they are not exist?

Reply:
addon controller will take the addon spec and just generate HelmChart crd, and in the same namespace as the addon object. There is no easy way to have namespace exist before addon is created.

We could introduce a targetNamespace in addon, which would map to HelmChart specs targetNamespace and createNamespace fields:

https://github.com/k3s-io/helm-controller/blob/master/pkg/apis/helm.cattle.io/v1/types.go#L22-L23

Reply:
Let's file an issue to track it as a new enhancement, thanks.